stdlib/2/typing.pyi

```diff
@@ -16,6 +16,7 @@ Type = object()
 _promote = object()
 no_type_check = object()
 ClassVar = object()
+Protocol = object()
```

Comment:
Oh I see, I thought it would just ignore the implementation in the stub. Ignore this then.